examples/aa-simple-dapp/src/components/Footer.tsx

```diff
@@ -16,7 +16,7 @@ export default function Footer() {
       />
       <FooterCard
         title={"Alchemy.com"}
-        subTitle="Check out other Alchemy products!"
+        subTitle="Check out our other products!"
```

Comment:
let's keep this as Alchemy since it's on the example app, not docs

Reply:
done